src/components/Leaderboards/MonthlyContribution.re

```diff
@@ -118,48 +119,50 @@ let make = (~numberOfLeaders) => {
         {switch (topContributorsOpt) {
          | Some(topContributors) =>
            ReasonReact.array(
-             Array.mapi(
+             topContributors->Array.mapWithIndex(
```

Comment:
No changes in this code, just switched to use the "Belt" `Array', rather than the default `Array`.